backend/tracim_backend/lib/proxy/proxy.py

```diff
@@ -39,24 +39,28 @@ def __init__(
         base_address: str,
         default_request_header_to_drop: typing.List[str] = DEFAULT_REQUEST_HEADER_TO_DROP,
         default_response_header_to_drop: typing.List[str] = DEFAULT_RESPONSE_HEADER_TO_DROP,
+        auth: typing.Optional[tuple] = None
```

Comment:
"optionnal tuple" does not help. What is the expected content?

Reply:
by the way, a list of headers to drop  must be named correctly in the plural form. Eg `default_request_headers_to_drop` instead of `default_request_header_to_drop`

Reply:
@lebouquetin Expected content is content following the spec of auth parameter of request of request lib. I didn't see much information than it want optional tuple, internal request lib check is about tuple or None.

Reply:
@lebouquetin plurals forms of "headers" parameters fix.

Reply:
maybe, should we decide, that we allow only str : `auth: typing.Optional[typing.Tuple[str, str]]` ?

Reply:
something like this: `auth: typing.Optional[typing.Tuple[str, str]]  # example: [login, password]`